tools/wasme/cli/example/cpp-istio-1.7/.bazelversion

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
This is a really old version, these all still work?

Reply:
o copied it from upstream:
https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/7afb39d868a973caa6216a535c24e37fb666b6f3/.bazelversion#L1